src/script/standard.h

```diff
@@ -40,7 +40,7 @@ extern unsigned nMaxDatacarrierBytes;
  * Failing one of these tests may trigger a DoS ban - see CheckInputs() for
  * details.
  */
-static const unsigned int MANDATORY_SCRIPT_VERIFY_FLAGS = SCRIPT_VERIFY_P2SH | SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY | SCRIPT_VERIFY_WITHDRAW;
```

Comment:
I believe you can remove this constant (``SCRIPT_VERIFY_WITHDRAW``) all together: https://github.com/ElementsProject/elements/blob/elements-0.14.1/src/script/interpreter.h#L113